tests/main.cpp

```diff
@@ -0,0 +1,10 @@
+// Renames main() in main.cpp to make room for the unit test main().
+#define main SunshineMain
+
+#include "gtest/gtest.h"
+
+int
+main(int argc, char **argv) {
+  ::testing::InitGoogleTest(&argc, argv);
+  return RUN_ALL_TESTS();
+}
```

Comment:
Test binary actually fails (at least on Windows).

```txt
sunshine_tests.exe --gtest_color=no
Testing started at 10:03 AM ...
[2024-02-05 10:03:19.303794] [0x00007dbc] [info]    Failed to apply config: filesystem error: cannot copy file: No such file or directory [assets/apps.json] [C:\Users\ReenigneArcher\Documents\GitHub\LizardByte\Sunshine\cmake-build-release\tests\config/apps.json]
[2024-02-05 10:03:19.303817] [0x00007dbc] [info]    To relaunch Sunshine successfully, use the shortcut in the Start Menu. Do not run Sunshine.exe manually.
Process finished with exit code 0
```

How to resolve this?

Reply:
Must remove `main.cpp` from test executable. This will require some refactoring of main.cpp.